run-nitro-test-node/action.yml

```diff
@@ -11,19 +11,19 @@ runs:
     - name: Checkout
       uses: actions/checkout@v3
       with:
-        repository: OffchainLabs/nitro
+        repository: OffchainLabs/nitro-testnode
         submodules: 'recursive'
```

Comment:
not needed. This unnecessarily checks out blockscout.

Reply:
Seems like there are some issues with this parameter: https://github.com/actions/checkout/issues/988

I'll just keep it as `true` for now.